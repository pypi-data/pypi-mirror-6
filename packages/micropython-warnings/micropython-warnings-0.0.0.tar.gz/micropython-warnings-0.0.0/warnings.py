# MicroPython doesn't support warnings
