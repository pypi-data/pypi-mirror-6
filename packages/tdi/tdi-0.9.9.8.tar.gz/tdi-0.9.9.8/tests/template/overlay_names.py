#!/usr/bin/env python
import warnings as _warnings
_warnings.resetwarnings()
_warnings.filterwarnings('error')

from tdi import html

template = html.from_string("""
<node tdi="item">
    <znode tdi:overlay="foo">
        <ynode tdi:overlay="bar"></ynode>
        <ynode tdi:overlay="<zonk"></ynode>
        <ynode tdi:overlay=">plenk"></ynode>
    </znode>
    <xnode tdi:overlay=">baz"></xnode>
</node>
""".lstrip())

print list(sorted(template.source_overlay_names))
print list(sorted(template.target_overlay_names))
