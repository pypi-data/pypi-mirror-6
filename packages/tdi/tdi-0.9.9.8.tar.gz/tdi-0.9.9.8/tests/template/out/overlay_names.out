['bar', 'foo', 'zonk']
['bar', 'baz', 'foo', 'plenk']
